metrics/registry.go

```diff
@@ -250,3 +278,10 @@ func (registry *registryImpl) Poll() *metrics_pb.MetricsMessage {
 
 	return (*metrics_pb.MetricsMessage)(builder)
 }
+
+type refCounted interface {
```

Comment:
why not put this interface into the util/concurrenz package and make it public?

Reply:
nevermind - it's a type in there already ... somehow that didn't register even though i just looked at it